include/zephyr/device.h

```diff
@@ -1156,7 +1179,7 @@ device_get_dt_nodelabels(const struct device *dev)
 		.api = (api_),								\
 		.state = (state_),							\
 		.data = (data_),							\
-		.ops = { .init = (init_fn_), .deinit = (deinit_fn_) },			\
+		.ops = Z_DEVICE_OPS(init_fn_, deinit_fn_),				\
```

Comment:
We could avoid extra macro by
```
.ops = { .init = (init_fn_), \
	IF_ENABLED(CONFIG_DEVICE_PROVIDE_DEINIT, (.deinit = (deinit_fn_)) \
}, \
```

Reply:
this config option relates to struct device_ops  so I prefer to clearly not using it for the macro that initialize struct device